bin/setup

```diff
@@ -40,7 +40,7 @@ FileUtils.chdir APP_ROOT do
   #
   # Validate that the ruby version requirement is met.
   #
-  expected_ruby_version = `cat .ruby-version`.chomp
+  expected_ruby_version = `cat .ruby-version`.chomp.gsub(/\.\d+\z/, "")
```

Comment:
This allows devcontainer to use 3.2.7 while everything else is still on 3.2.2. We can upgrade everything else later, but this chopping off of the minor version should be pretty safe.